src/Aspire.Dashboard/Extensions/ResourceViewModelExtensions.cs

```diff
@@ -7,11 +7,6 @@ namespace Aspire.Dashboard.Extensions;
 
 internal static class ResourceViewModelExtensions
 {
-    public static bool IsHiddenState(this ResourceViewModel resource)
```

Comment:
I think we keep this for now so hidden resources don't suddenly start showing up. We can still infer hidden from this.

Reply:
Added a method `IsHidden` that also checks for `Hidden` state.